R/build-home.R

```diff
@@ -262,13 +269,28 @@ cran_mirror <- function() {
     cran
   }
 }
-on_cran <- function(pkg, cran = cran_mirror()) {
-  pkgs <- utils::available.packages(
-    type = "source",
-    contriburl = paste0(cran, "/src/contrib"))
-  pkg %in% rownames(pkgs)
+
+bioc_mirror <- function() {
+  if (requireNamespace("BiocInstaller", quietly = TRUE)) {
+    bioc <-
+      utils::contrib.url(BiocInstaller::biocinstallRepos()[["BioCsoft"]])
+  } else {
+    bioc <-
+      "https://bioconductor.org/packages/3.5/bioc/src/contrib"
+  }
+  bioc
 }
 
+in_repo <- function(pkg, cran = cran_mirror(), bioc = bioc_mirror()) {
```

Comment:
Why don't we just make this `repo_url`, returning `NULL` if it's not in any repo